src/content/blog/you-probably-don't-need-eslint-config-prettier-or-eslint-plugin-prettier/index.mdx

```diff
@@ -0,0 +1,212 @@
+Their downside was that they were oftentimes too opinionated - even enabling formatting rules.
+
+Developers got around those formatting rules by knowing that ESLint evaluates configs in the order they're listed under `"extends"`.
+`eslint-config-prettier` could be placed last in a project's ESLint config to turn off any formatting rules enabled by previous plugins.
+
+```json
+{
+	"extends": [
+		// 1. Configures many ESLint rules, including enabling some formatting ones
+		"airbnb",
+		// 2. Disables only the formatting rules from previous plugins
+		"prettier"
+	]
+}
+```
+
+By extending from `eslint-config-prettier` last, projects could gain the benefits of those popular shared configs without having to run formatting rules within ESLint.
+
+### Why `eslint-config-prettier` Is Often Unnecessary
+
+Over the last few years, ESLint best practices have advanced in two ways (among others):
+
+- ESLint core and most community plugins have determined that enabling overly opinionated rules -especially stylistic ones- in shared configs makes developers dislike ESLint without much real benefit
+- The recommended rulesets in ESLint and typescript-eslint have grown to include most of the beneficial logical rules that shared configs such as `eslint-config-airbnb` were largely used for
+
+As a result, many new projects haven't felt a need to load in opinionated configs such as `eslint-config-airbnb`.
+Many projects start out with a much more straightforward set of configs:
+
+1. To start: `"eslint:recommended"`, ESLint's built-in recommended configuration
+2. If using TypeScript: `"plugin:@typescript-eslint/recommended"` or `"plugin:@typescript-eslint/recommended-type-checked"` for the recommended TypeScript rules
+3. Any framework- or library-specific plugins, such as [`eslint-plugin-jsx-a11y`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y)'s `"plugin:jsx-a11y/recommended"`
+
+**If you don't use a legacy ESLint shared config that enables formatting rules, you probably don't need `eslint-config-prettier`.**
+Adding `eslint-config-prettier` at the end of the `"extends"` list doesn't do anything if nothing enabled formatting rules to begin with.
+Therefore, most projects don't gain any benefit from `eslint-config-prettier`.
+
+Furthermore, two insidious points of confusing can arise from using `eslint-config-prettier` redundantly:
+
+- Seeing a reference to _"prettier"_ in an ESLint configuration can confuse developers new to the area.
+- Nothing stops projects from manually re-enabling formatting rules under the ESLint configuration's `"overrides"` or `"rules"` properties.
+
+I now recommend against including `eslint-config-prettier` in most new projects.
+
+> 💡 Not sure whether you can safely remove `"prettier"` from your `"extends"`?
+> Try removing it, running ESLint with [`--print-config`](https://eslint.org/docs/latest/use/command-line-interface#--print-config), and seeing which lint rules are newly enabled.
+
+## `eslint-plugin-prettier`
```

Comment:
[Todo] I should reach out to the maintainers of this package to make sure I'm not misrepresenting it or otherwise being rude.

Reply:
I've reached out. Will keep this open as a reminder that I should give them each a business week to get back to me.

Reply:
I reached out privately. Shoutout https://github.com/JoshuaKGoldberg/github-username-to-emails & https://github.com/JoshuaKGoldberg/github-username-to-emails-site for saving time finding emails...

Since this is the last third of December, I'll ping folks again the first week of January. Just in case they're already out.